### PR TITLE
85589 update insurance flow 10 70959c

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR updates the insurance flow along with some copy.

- Updates titles to drop "primary" and "secondary"
- Reorganizes a couple pages so they show up earlier in the form flow
- Adds some unit test coverage for the statement of truth portion of the form.
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/85589

## Testing done

- Manual
- Unit

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Primary  |Screenshot 2024-06-20 at 11 15 44|Screenshot 2024-06-20 at 11 12 25|
| Secondary |Screenshot 2024-06-20 at 11 16 02|Screenshot 2024-06-20 at 11 12 48|
| Unit test coverage |Screenshot 2024-06-20 at 11 16 08|Screenshot 2024-06-20 at 11 07 30|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function applicantHasInsuranceSchema(isPrimary) {
       ...titleUI(
         ({ formData }) =>
           `${nameWording(formData, undefined, undefined, true)} ${
-            isPrimary ? 'primary' : 'secondary'
+            isPrimary ? '' : 'additional'
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js:54]: We drop primary/secondary label and just use blank for primary insurance and 'additional' for secondary. This is done in a couple places.

File: src/applications/ivc-champva/10-7959C/config/form.js [hunk 2]
```diff
@@ -233,9 +233,29 @@ const formConfig = {
...
+          path: 'insurance-medigap',
+          depends: formData =>
+            get('applicantHasPrimary', formData) &&
+            get('applicantPrimaryInsuranceType', formData) === 'medigap',
+          title: formData =>
+            `${fnp(formData)} ${
+              formData.applicantPrimaryProvider
+            } Medigap information`,
+          ...applicantMedigapSchema(true),
+        },
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/config/form.js:258]: Moved these pages up earlier in the flow

File: src/applications/ivc-champva/10-7959C/tests/config/form.unit.spec.jsx [hunk 3]
```diff
@@ -340,3 +340,12 @@ describe('FileFieldWrapped', () => {
...
 });
+
+testNumberOfWebComponentFields(
+  formConfig,
+  formConfig.chapters.formSignature.pages.formSignature.schema,
+  formConfig.chapters.formSignature.pages.formSignature.uiSchema,
+  1,
+  'Form signature page',
+  { ...mockData.data },
+);
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/tests/config/form.unit.spec.jsx:351]: Unrelated, but added unit test verifying the "who's signing this document" page adds the right number of web components to the page.

File: src/applications/ivc-champva/10-7959C/tests/containers/ConfirmationPage.unit.spec.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/tests/containers/ConfirmationPage.unit.spec.js:1]: Unrelated to the issue at hand, but increase unit test coverage for the "statement of truth" display. Nothing crazy, just bumping coverage up a bit.

**Review by cloudmagic80 — APPROVED** [review]

**Review by reiting — APPROVED** [review]

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**